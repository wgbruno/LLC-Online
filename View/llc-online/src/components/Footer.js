import React from 'react'; 
import { SocialIcon } from 'react-social-icons';

export default function Footer(){
    return (
        <div>
        <footer id="footer">
          <div className="footer-top">
            <div className="container">
              <div className="row">
                <div className="col-lg-3 col-md-6 footer-contact">
                  <h3>LLC</h3>
                  <p>
                    Maryland Hall 443 <br />
                    Loyola University Maryland<br />
                    Baltimore MD 21210 <br /><br />
                    <strong>Phone:</strong> 410-617-2674<br />
                    <strong>Email:</strong> loyolallc@loyola.edu<br />
                  </p>
                </div>
                <div className="col-lg-3 col-md-6 footer-links">
                  <h4>Useful Links</h4>
                  <ul>
                    <li><i className="bx bx-chevron-right" /> <a href="/">Home</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="/login">Login</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="/calendar">Calendar</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="/inventory">Inventory</a></li>
                    <i className="bx bx-chevron-right" /> <a href="/contact">Contact Us</a>
                  </ul>
                </div>
                {/*<div className="col-lg-3 col-md-6 footer-links">
                  <h4>Our Services</h4>
                  <ul>
                    <li><i className="bx bx-chevron-right" /> <a href="#">Web Design</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="#">Web Development</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="#">Product Management</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="#">Marketing</a></li>
                    <li><i className="bx bx-chevron-right" /> <a href="#">Graphic Design</a></li>
                  </ul>
                </div>*/}
                <div className="col-lg-3 col-md-6 footer-links">
                  <h4>Our Social Networks</h4>
                  <div className="social-links mt-3">
                    <SocialIcon url="https://twitter.com/LoCo_LLC" />
                    <SocialIcon url="https://www.instagram.com/loyolallc/" />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div className="container py-4">
            <div className="copyright">
              Â© Copyright <strong><span>Butterfly</span></strong>. All Rights Reserved
            </div>
            <div className="credits">
              All the links in the footer should remain intact.
              You can delete the links only if you purchased the pro version. */
              Licensing information: https://bootstrapmade.com/license/ */
              Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/butterfly-free-bootstrap-theme/
              Designed by <a href="https://bootstrapmade.com/">BootstrapMade</a>
            </div>
          </div>
        </footer>{/* End Footer */}
      </div>
    )

}


