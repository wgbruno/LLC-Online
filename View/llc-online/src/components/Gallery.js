import React from 'react'; 

export default function Gallery(){
    return (
        <div>
        {/* ======= Gallery Section ======= */}
      <section id="gallery" className="gallery">
      <div className="container">
        <div className="section-title">
          <h2>Gallery</h2>
          <p></p>
        </div>
        <div className="row no-gutters">
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/all-groups-B2.jpg")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/all-groups-B2.jpg")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/crepes 1.jpg")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/crepes 1.jpg")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/Copy of Student of the Month poster.png")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/Copy of Student of the Month poster.png")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/Untitled design.png")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/Untitled design.png")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/LEAP 2.jpg")}className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/LEAP 2.jpg")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/Composition Workshop.png")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/Composition Workshop.png")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/LLC Staff pick movie nights.png")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/LLC Staff pick movie nights.png")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
          <div className="col-lg-3 col-md-4">
            <div className="gallery-item">
              <a href={require("../assets/Teatime3.jpg")} className="galleery-lightbox" data-gallery="gallery-item">
                <img src={require("../assets/Teatime3.jpg")} alt="" className="img-fluid" />
              </a>
            </div>
          </div>
        </div>
      </div>
    </section>
    </div>
    )
}