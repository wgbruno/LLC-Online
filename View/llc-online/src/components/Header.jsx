import React from 'react'; 

export default function Header(){
    return (
            <div>
            {/*<!-- ======= Header ======= -->*/}
            <header id="header" class="fixed-top">
            <div class="container d-flex align-items-center justify-content-between">

                <a href="index.html" class="logo"><img src="assets/img/logo.png" alt="" class="img-fluid" /></a>
                {/*<!-- Uncomment below if you prefer to use text as a logo -->}
                <!-- <h1 class="logo"><a href="index.html">Butterfly</a></h1> -->*/}
            </div>
            </header>{/*<!-- End Header -->*/}
            </div>
    )

}