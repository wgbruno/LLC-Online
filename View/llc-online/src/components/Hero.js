import React, { Component } from 'react'; 

export default function Hero(){
        return (
            <div>
            {/* ======= Hero Section ======= */}
                <div className="container">
                    <div className="row">
                    <div className="col-lg-6 pt-4 pt-lg-0 order-2 order-lg-1 d-flex flex-column justify-content-center text-center">
                        <h1>The Language Learning Center Online</h1>
                        {/*<p>The Language Learning Center is a multimedia resource center dedicated to supporting language, 
                            literature, and cultural studies at Loyola. 
                            Our facilities include a state of the art computer lab, two video viewing rooms, 
                            an extensive foreign language film collection, and a comfortable reading room perfect 
                            for independent study as well as group projects. The LLC, along with its friendly staff, 
                            serves as the technology hub for the Department of Modern Languages and Literatures, 
                            providing both the equipment and support to bring language studies to life 
                            using the latest technology.</p>*/}
                    </div>
                    <div className="col-lg-6 order-1 order-lg-2 hero-img">
                        <img src={require("../assets/rsz_standard.jpg")} className="img-fluid" alt="" />
                    </div>
                    </div>
                </div>
            {/* End Hero */}
            </div>
    )
}