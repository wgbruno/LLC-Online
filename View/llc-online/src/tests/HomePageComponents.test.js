import {expect, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import Header from "../components/Header"
import Footer from "../components/Footer"
import Hero from "../components/Hero"
import NavBar from "../components/NavBar"
import Gallery from "../components/Gallery"

test("Header", function (){
    const header = renderer.create(<Header />).toJSON();
    expect(header).toMatchSnapshot();
});

test("Footer", function (){
    const footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot();
});

test("Hero", function (){
    const hero = renderer.create(<Hero />).toJSON();
    expect(hero).toMatchSnapshot();
});

test("NavBar", function (){
    const navbar = renderer.create(<NavBar />).toJSON();
    expect(navbar).toMatchSnapshot();
});

test("Gallery", function (){
    const gallery = renderer.create(<Gallery />).toJSON();
    expect(gallery).toMatchSnapshot();
});