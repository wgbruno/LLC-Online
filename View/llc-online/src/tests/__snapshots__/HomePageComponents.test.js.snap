// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<div>
  <footer
    id="footer"
  >
    <div
      className="footer-top"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-lg-3 col-md-6 footer-contact"
          >
            <h3>
              LLC
            </h3>
            <p>
              Maryland Hall 443 
              <br />
              Loyola University Maryland
              <br />
              Baltimore MD 21210 
              <br />
              <br />
              <strong>
                Phone:
              </strong>
               410-617-2674
              <br />
              <strong>
                Email:
              </strong>
               loyolallc@loyola.edu
              <br />
            </p>
          </div>
          <div
            className="col-lg-3 col-md-6 footer-links"
          >
            <h4>
              Useful Links
            </h4>
            <ul>
              <li>
                <i
                  className="bx bx-chevron-right"
                />
                 
                <a
                  href="/"
                >
                  Home
                </a>
              </li>
              <li>
                <i
                  className="bx bx-chevron-right"
                />
                 
                <a
                  href="/login"
                >
                  Login
                </a>
              </li>
              <li>
                <i
                  className="bx bx-chevron-right"
                />
                 
                <a
                  href="/calendar"
                >
                  Calendar
                </a>
              </li>
              <li>
                <i
                  className="bx bx-chevron-right"
                />
                 
                <a
                  href="/inventory"
                >
                  Inventory
                </a>
              </li>
              <i
                className="bx bx-chevron-right"
              />
               
              <a
                href="mailtoloyolallc@loyola.edu"
              >
                Contact Us
              </a>
            </ul>
          </div>
          <div
            className="col-lg-3 col-md-6 footer-links"
          >
            <h4>
              Our Social Networks
            </h4>
            <div
              className="social-links mt-3"
            >
              <a
                aria-label="twitter"
                className="social-icon"
                href="https://twitter.com/LoCo_LLC"
                style={
                  Object {
                    "display": "inline-block",
                    "height": "50px",
                    "overflow": "hidden",
                    "position": "relative",
                    "verticalAlign": "middle",
                    "width": "50px",
                  }
                }
              >
                <div
                  className="social-container"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "position": "absolute",
                      "top": 0,
                      "width": "100%",
                    }
                  }
                >
                  <svg
                    className="social-svg"
                    style={
                      Object {
                        "borderRadius": "50%",
                        "fillRule": "evenodd",
                        "height": "100%",
                        "left": 0,
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                    viewBox="0 0 64 64"
                  >
                    <g
                      className="social-svg-background"
                      style={
                        Object {
                          "MozTransition": "fill 170ms ease-in-out",
                          "OTransition": "fill 170ms ease-in-out",
                          "WebkitTransition": "fill 170ms ease-in-out",
                          "fill": "transparent",
                          "msTransition": "fill 170ms ease-in-out",
                          "transition": "fill 170ms ease-in-out",
                        }
                      }
                    >
                      <circle
                        cx="32"
                        cy="32"
                        r="31"
                      />
                    </g>
                    <g
                      className="social-svg-icon"
                      style={
                        Object {
                          "MozTransition": "fill 170ms ease-in-out",
                          "OTransition": "fill 170ms ease-in-out",
                          "WebkitTransition": "fill 170ms ease-in-out",
                          "fill": "transparent",
                          "msTransition": "fill 170ms ease-in-out",
                          "transition": "fill 170ms ease-in-out",
                        }
                      }
                    >
                      <path
                        d="m 48.1,22.1 c -1.223105,0.430685 -2.4,0.761371 -3.8,1 1.4,-0.8 2.4,-2.1 2.9,-3.6 -1.3,0.8 -2.7,1.3 -4.2,1.6 -1.2,-1.3 -3,-2.1 -4.8,-2.1 -3.6,0 -6.6,2.9 -6.6,6.6 0,0.5 0.1,1 0.2,1.5 -5.5,-0.3 -10.3,-2.9 -13.5,-6.9 -0.6,1 -0.9,2.1 -0.9,3.3 0,2.3 1.2,4.3 2.9,5.5 -1.1,0 -2.1,-0.3 -3,-0.8 v 0.1 c 0,3.2 2.3,5.8 5.3,6.4 -0.6,0.1 -1.1,0.2 -1.7,0.2 -0.4,0 -0.8,0 -1.2,-0.1 0.8,2.6 3.3,4.5 6.1,4.6 -2.2,1.8 -5.1,2.8 -8.2,2.8 -0.5,0 -1.1,0 -1.6,-0.1 C 18.9,44 22.4,45 26.1,45 38.2,45 44.43575,34.996634 44.67774,26.3 L 44.7,25.5 c 1.2,-1 2.5,-2.1 3.4,-3.4 z"
                      />
                    </g>
                    <g
                      className="social-svg-mask"
                      style={
                        Object {
                          "MozTransition": "fill 170ms ease-in-out",
                          "OTransition": "fill 170ms ease-in-out",
                          "WebkitTransition": "fill 170ms ease-in-out",
                          "fill": "#00aced",
                          "msTransition": "fill 170ms ease-in-out",
                          "transition": "fill 170ms ease-in-out",
                        }
                      }
                    >
                      <path
                        d="M0,0v64h64V0H0z M44.7,25.5c0,0.3,0,0.6,0,0.8C44.7,35,38.1,45,26.1,45c-3.7,0-7.2-1.1-10.1-2.9 c0.5,0.1,1,0.1,1.6,0.1c3.1,0,5.9-1,8.2-2.8c-2.9-0.1-5.3-2-6.1-4.6c0.4,0.1,0.8,0.1,1.2,0.1c0.6,0,1.2-0.1,1.7-0.2 c-3-0.6-5.3-3.3-5.3-6.4c0,0,0-0.1,0-0.1c0.9,0.5,1.9,0.8,3,0.8c-1.8-1.2-2.9-3.2-2.9-5.5c0-1.2,0.3-2.3,0.9-3.3 c3.2,4,8.1,6.6,13.5,6.9c-0.1-0.5-0.2-1-0.2-1.5c0-3.6,2.9-6.6,6.6-6.6c1.9,0,3.6,0.8,4.8,2.1c1.5-0.3,2.9-0.8,4.2-1.6 c-0.5,1.5-1.5,2.8-2.9,3.6c1.3-0.2,2.6-0.5,3.8-1C47.1,23.4,46,24.5,44.7,25.5z"
                      />
                    </g>
                  </svg>
                </div>
              </a>
              <a
                aria-label="instagram"
                className="social-icon"
                href="https://www.instagram.com/loyolallc/"
                style={
                  Object {
                    "display": "inline-block",
                    "height": "50px",
                    "overflow": "hidden",
                    "position": "relative",
                    "verticalAlign": "middle",
                    "width": "50px",
                  }
                }
              >
                <div
                  className="social-container"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "position": "absolute",
                      "top": 0,
                      "width": "100%",
                    }
                  }
                >
                  <svg
                    className="social-svg"
                    style={
                      Object {
                        "borderRadius": "50%",
                        "fillRule": "evenodd",
                        "height": "100%",
                        "left": 0,
                        "position": "absolute",
                        "top": 0,
                        "width": "100%",
                      }
                    }
                    viewBox="0 0 64 64"
                  >
                    <g
                      className="social-svg-background"
                      style={
                        Object {
                          "MozTransition": "fill 170ms ease-in-out",
                          "OTransition": "fill 170ms ease-in-out",
                          "WebkitTransition": "fill 170ms ease-in-out",
                          "fill": "transparent",
                          "msTransition": "fill 170ms ease-in-out",
                          "transition": "fill 170ms ease-in-out",
                        }
                      }
                    >
                      <circle
                        cx="32"
                        cy="32"
                        r="31"
                      />
                    </g>
                    <g
                      className="social-svg-icon"
                      style={
                        Object {
                          "MozTransition": "fill 170ms ease-in-out",
                          "OTransition": "fill 170ms ease-in-out",
                          "WebkitTransition": "fill 170ms ease-in-out",
                          "fill": "transparent",
                          "msTransition": "fill 170ms ease-in-out",
                          "transition": "fill 170ms ease-in-out",
                        }
                      }
                    >
                      <path
                        d="M 39.88,25.89 C 40.86,25.89 41.65,25.10 41.65,24.12 41.65,23.14 40.86,22.35 39.88,22.35 38.90,22.35 38.11,23.14 38.11,24.12 38.11,25.10 38.90,25.89 39.88,25.89 Z M 32.00,24.42 C 27.82,24.42 24.42,27.81 24.42,32.00 24.42,36.19 27.82,39.58 32.00,39.58 36.18,39.58 39.58,36.18 39.58,32.00 39.58,27.82 36.18,24.42 32.00,24.42 Z M 32.00,36.92 C 29.28,36.92 27.08,34.72 27.08,32.00 27.08,29.28 29.28,27.08 32.00,27.08 34.72,27.08 36.92,29.28 36.92,32.00 36.92,34.72 34.72,36.92 32.00,36.92 Z M 32.00,19.90 C 35.94,19.90 36.41,19.92 37.96,19.99 39.41,20.05 40.19,20.29 40.71,20.50 41.40,20.77 41.89,21.08 42.41,21.60 42.92,22.12 43.24,22.61 43.51,23.30 43.71,23.82 43.95,24.60 44.02,26.04 44.09,27.60 44.11,28.06 44.11,32.01 44.11,35.95 44.09,36.41 44.02,37.97 43.95,39.41 43.71,40.19 43.51,40.71 43.24,41.40 42.92,41.90 42.41,42.41 41.89,42.93 41.40,43.25 40.71,43.51 40.19,43.71 39.41,43.96 37.96,44.02 36.41,44.09 35.94,44.11 32.00,44.11 28.06,44.11 27.59,44.09 26.04,44.02 24.59,43.96 23.81,43.72 23.29,43.51 22.60,43.24 22.11,42.93 21.59,42.41 21.08,41.90 20.76,41.40 20.49,40.71 20.29,40.19 20.05,39.41 19.98,37.97 19.91,36.41 19.89,35.95 19.89,32.01 19.89,28.06 19.91,27.60 19.98,26.04 20.05,24.60 20.29,23.82 20.49,23.30 20.76,22.61 21.08,22.12 21.59,21.60 22.11,21.08 22.60,20.76 23.29,20.50 23.81,20.30 24.59,20.05 26.04,19.99 27.59,19.91 28.06,19.90 32.00,19.90 Z M 32.00,17.24 C 27.99,17.24 27.49,17.26 25.91,17.33 24.34,17.40 23.27,17.65 22.33,18.01 21.36,18.39 20.54,18.90 19.72,19.72 18.90,20.54 18.39,21.37 18.01,22.33 17.65,23.27 17.40,24.34 17.33,25.92 17.26,27.49 17.24,27.99 17.24,32.00 17.24,36.01 17.26,36.51 17.33,38.09 17.40,39.66 17.65,40.73 18.01,41.67 18.39,42.65 18.90,43.47 19.72,44.29 20.54,45.11 21.37,45.61 22.33,45.99 23.27,46.36 24.34,46.61 25.92,46.68 27.49,46.75 27.99,46.77 32.01,46.77 36.02,46.77 36.52,46.75 38.09,46.68 39.66,46.61 40.74,46.36 41.68,45.99 42.65,45.62 43.47,45.11 44.29,44.29 45.11,43.47 45.62,42.64 46.00,41.67 46.36,40.74 46.61,39.66 46.68,38.09 46.75,36.51 46.77,36.01 46.77,32.00 46.77,27.99 46.75,27.49 46.68,25.91 46.61,24.34 46.36,23.27 46.00,22.33 45.62,21.35 45.11,20.53 44.29,19.71 43.47,18.89 42.65,18.39 41.68,18.01 40.74,17.64 39.67,17.39 38.09,17.32 36.51,17.26 36.01,17.24 32.00,17.24 Z"
                      />
                    </g>
                    <g
                      className="social-svg-mask"
                      style={
                        Object {
                          "MozTransition": "fill 170ms ease-in-out",
                          "OTransition": "fill 170ms ease-in-out",
                          "WebkitTransition": "fill 170ms ease-in-out",
                          "fill": "#e94475",
                          "msTransition": "fill 170ms ease-in-out",
                          "transition": "fill 170ms ease-in-out",
                        }
                      }
                    >
                      <path
                        d="M0,0v64h64V0H0z M 42.03,23.99 C 42.03,24.99 41.22,25.79 40.23,25.79 39.23,25.79 38.43,24.99 38.43,23.99 38.43,22.99 39.23,22.19 40.23,22.19 41.22,22.19 42.03,22.99 42.03,23.99 Z M 24.52,31.99 C 24.52,27.74 27.97,24.29 32.22,24.29 36.47,24.29 39.92,27.75 39.92,31.99 39.92,36.24 36.47,39.70 32.22,39.70 27.97,39.70 24.52,36.25 24.52,31.99 Z M 27.22,31.99 C 27.22,34.76 29.46,36.99 32.22,36.99 34.98,36.99 37.22,34.76 37.22,31.99 37.22,29.23 34.98,27.00 32.22,27.00 29.46,27.00 27.22,29.23 27.22,31.99 Z M 38.28,19.79 C 36.70,19.72 36.22,19.70 32.22,19.70 28.22,19.70 27.74,19.71 26.17,19.79 24.69,19.85 23.90,20.11 23.37,20.31 22.67,20.58 22.17,20.90 21.65,21.43 21.13,21.96 20.80,22.46 20.53,23.16 20.33,23.68 20.08,24.48 20.01,25.94 19.94,27.52 19.92,27.99 19.92,32.01 19.92,36.01 19.94,36.48 20.01,38.06 20.08,39.52 20.33,40.32 20.53,40.84 20.80,41.54 21.13,42.05 21.65,42.57 22.17,43.10 22.67,43.41 23.37,43.69 23.90,43.90 24.69,44.15 26.17,44.21 27.74,44.28 28.22,44.30 32.22,44.30 36.22,44.30 36.70,44.28 38.28,44.21 39.75,44.15 40.54,43.89 41.07,43.69 41.77,43.42 42.27,43.10 42.80,42.57 43.32,42.05 43.64,41.54 43.91,40.84 44.12,40.32 44.36,39.52 44.43,38.06 44.50,36.48 44.52,36.01 44.52,32.01 44.52,27.99 44.50,27.52 44.43,25.94 44.36,24.48 44.12,23.68 43.91,23.16 43.64,22.46 43.32,21.96 42.80,21.43 42.27,20.90 41.77,20.59 41.07,20.31 40.54,20.10 39.75,19.85 38.28,19.79 Z M 26.03,17.09 C 27.64,17.02 28.15,17.00 32.22,17.00 36.27,17.00 36.80,17.02 38.38,17.08 38.38,17.08 38.41,17.08 38.41,17.08 40.01,17.15 41.10,17.41 42.06,17.78 43.04,18.17 43.87,18.68 44.71,19.51 45.54,20.34 46.06,21.18 46.44,22.17 46.81,23.13 47.06,24.21 47.14,25.81 47.21,27.41 47.23,27.92 47.23,31.99 47.23,36.07 47.21,36.58 47.14,38.18 47.06,39.78 46.81,40.87 46.44,41.82 46.06,42.80 45.54,43.65 44.71,44.48 43.87,45.31 43.04,45.83 42.06,46.21 41.10,46.58 40.00,46.84 38.41,46.91 36.81,46.98 36.31,47.00 32.23,47.00 28.15,47.00 27.64,46.98 26.04,46.91 24.44,46.84 23.35,46.58 22.40,46.21 21.42,45.82 20.58,45.31 19.75,44.48 18.91,43.65 18.39,42.81 18.01,41.82 17.64,40.86 17.39,39.78 17.32,38.18 17.25,36.58 17.23,36.07 17.23,31.99 17.23,27.92 17.25,27.41 17.32,25.82 17.39,24.21 17.64,23.13 18.01,22.17 18.39,21.20 18.91,20.35 19.75,19.52 20.58,18.69 21.41,18.17 22.40,17.78 23.35,17.42 24.44,17.16 26.03,17.09 Z"
                      />
                    </g>
                  </svg>
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="container py-4"
    >
      <div
        className="copyright"
      >
        © Copyright 
        <strong>
          <span>
            Butterfly
          </span>
        </strong>
        . All Rights Reserved
      </div>
      <div
        className="credits"
      >
        All the links in the footer should remain intact. You can delete the links only if you purchased the pro version. */ Licensing information: https://bootstrapmade.com/license/ */ Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/butterfly-free-bootstrap-theme/ Designed by 
        <a
          href="https://bootstrapmade.com/"
        >
          BootstrapMade
        </a>
      </div>
    </div>
  </footer>
</div>
`;

exports[`Gallery 1`] = `
<div>
  <section
    className="gallery"
    id="gallery"
  >
    <div
      className="container"
    >
      <div
        className="section-title"
      >
        <h2>
          Gallery
        </h2>
        <p>
          Magnam dolores commodi suscipit. Necessitatibus eius consequatur ex aliquid fuga eum quidem. Sit sint consectetur velit. Quisquam quos quisquam cupiditate. Et nemo qui impedit suscipit alias ea. Quia fugiat sit in iste officiis commodi quidem hic quas.
        </p>
      </div>
      <div
        className="row no-gutters"
      >
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-1.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-1.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-2.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-2.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-3.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-3.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-4.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-4.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-5.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-5.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-6.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-6.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-7.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-7.jpg"
              />
            </a>
          </div>
        </div>
        <div
          className="col-lg-3 col-md-4"
        >
          <div
            className="gallery-item"
          >
            <a
              className="galleery-lightbox"
              data-gallery="gallery-item"
              href="assets/img/gallery/gallery-8.jpg"
            >
              <img
                alt=""
                className="img-fluid"
                src="assets/img/gallery/gallery-8.jpg"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Header 1`] = `
<div>
  <header
    class="fixed-top"
    id="header"
  >
    <div
      class="container d-flex align-items-center justify-content-between"
    >
      <a
        class="logo"
        href="index.html"
      >
        <img
          alt=""
          class="img-fluid"
          src="assets/img/logo.png"
        />
      </a>
    </div>
  </header>
</div>
`;

exports[`Hero 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-6 pt-4 pt-lg-0 order-2 order-lg-1 d-flex flex-column justify-content-center"
      >
        <h1>
          The Language Learning Center Online
        </h1>
        <h2>
          Insert phrase here about the LLC
        </h2>
      </div>
      <div
        className="col-lg-6 order-1 order-lg-2 hero-img"
      >
        <img
          alt=""
          className="img-fluid"
          src="assets/img/hero-img.png"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`NavBar 1`] = `
<div>
  <section
    className="services section-bg"
    id="services"
  >
    <div
      className="container"
    >
      <div
        className="section-title"
      >
        <p>
          Magnam dolores commodi suscipit. Necessitatibus eius consequatur ex aliquid fuga eum quidem. Sit sint consectetur velit. Quisquam quos quisquam cupiditate. Et nemo qui impedit suscipit alias ea. Quia fugiat sit in iste officiis commodi quidem hic quas.
        </p>
      </div>
      <div
        className="row"
      >
        <div
          className="col-lg-4 col-md-6"
        >
          <div
            className="icon-box"
          >
            <div
              className="icon"
            >
              <i
                className="bi bi-cash-stack"
                style={
                  Object {
                    "color": "#ff689b",
                  }
                }
              />
            </div>
            <h4
              className="title"
            >
              <a
                href={true}
              >
                Lorem Ipsum
              </a>
            </h4>
            <p
              className="description"
            >
              Voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-md-6"
        >
          <div
            className="icon-box"
          >
            <div
              className="icon"
            >
              <i
                className="bi bi-calendar4-week"
                style={
                  Object {
                    "color": "#e9bf06",
                  }
                }
              />
            </div>
            <h4
              className="title"
            >
              <a
                href={true}
              >
                Dolor Sitema
              </a>
            </h4>
            <p
              className="description"
            >
              Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat tarad limino ata
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-md-6"
          data-wow-delay="0.1s"
        >
          <div
            className="icon-box"
          >
            <div
              className="icon"
            >
              <i
                className="bi bi-chat-text"
                style={
                  Object {
                    "color": "#3fcdc7",
                  }
                }
              />
            </div>
            <h4
              className="title"
            >
              <a
                href={true}
              >
                Sed ut perspiciatis
              </a>
            </h4>
            <p
              className="description"
            >
              Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-md-6"
          data-wow-delay="0.1s"
        >
          <div
            className="icon-box"
          >
            <div
              className="icon"
            >
              <i
                className="bi bi-credit-card-2-front"
                style={
                  Object {
                    "color": "#41cf2e",
                  }
                }
              />
            </div>
            <h4
              className="title"
            >
              <a
                href={true}
              >
                Magni Dolores
              </a>
            </h4>
            <p
              className="description"
            >
              Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-md-6"
          data-wow-delay="0.2s"
        >
          <div
            className="icon-box"
          >
            <div
              className="icon"
            >
              <i
                className="bi bi-globe"
                style={
                  Object {
                    "color": "#d6ff22",
                  }
                }
              />
            </div>
            <h4
              className="title"
            >
              <a
                href={true}
              >
                Nemo Enim
              </a>
            </h4>
            <p
              className="description"
            >
              At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque
            </p>
          </div>
        </div>
        <div
          className="col-lg-4 col-md-6"
          data-wow-delay="0.2s"
        >
          <div
            className="icon-box"
          >
            <div
              className="icon"
            >
              <i
                className="bi bi-clock"
                style={
                  Object {
                    "color": "#4680ff",
                  }
                }
              />
            </div>
            <h4
              className="title"
            >
              <a
                href={true}
              >
                Eiusmod Tempor
              </a>
            </h4>
            <p
              className="description"
            >
              Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
